packages/action-merge/src/components/MergeWizard/MergeResponse.spec.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import MergeResponse from './MergeResponse'
+import {EntityResponseTable, PermissionMessageBox} from './MergeResponse'
+import {mount, render, shallow} from 'enzyme'
+import {FormattedMessage} from 'react-intl'
+
+const intl = {
```

Comment:
There is a file called `intlStub.js` in the `login` package which contains all required methods (`formatDate`, `formatTime`, ...). Maybe you could move this file somewhere else so that it can be used here too.

The `intl` prop type in `<MergeResponse>` should be `intlShape.isRequired` instead of `PropTypes.object`.